### PR TITLE
Rename `jvm.classes.current_loaded` to `jvm.classes.count`

### PR DESCRIPTION
**Pull request by trask**
Based on https://github.com/open-telemetry/opentelemetry-specification/issues/3429#issuecomment-1544344173

### CODE REVIEW

**Review by mateuszrzeszutek — APPROVED** [review]

**Review by joaopgrassi — APPROVED** [review]
Approving to get this moving (as it aligns with our current guidelines) but as @trask said we will need to revisit once #50 is solved.

**Review by donbourne — APPROVED** [review]

**Review by jack-berg — APPROVED** [review]

File: semantic_conventions/metrics/process-runtime-jvm-metrics.yaml [hunk 1]
```diff
@@ -101,6 +101,13 @@ groups:
         type: boolean
         requirement_level: recommended
 
+  - id: metric.process.runtime.jvm.classes.count
+    type: metric
+    metric_name: process.runtime.jvm.classes.count
+    brief: "Number of classes currently loaded."
+    instrument: updowncounter
+    unit: "{class}"
+
```
[the remark this answers is not in the document]
Comment (trask) [reply semantic_conventions/metrics/process-runtime-jvm-metrics.yaml:110]: thx! I sent #110

**Review by reyang — APPROVED** [review]

**Comment by jsuereth** [comment]
This needs to be updated for merge conflicts *and* I'd suggest thinking about the rename from `runtime.process.jvm` => `jvm` prior to this PR.

**Comment by trask** [comment]
> I'd suggest thinking about the rename from `runtime.process.jvm` => `jvm` prior to this PR.

I'm very supportive of this rename. I opened #227 to track.

**Review by mateuszrzeszutek — APPROVED** [review]

**Comment by trask** [comment]
@open-telemetry/specs-semconv-maintainers this is ready to merge

**Review by jack-berg — APPROVED** [review]

**Review by jsuereth — APPROVED** [review]

**Pull request merged**